### ISSUE TITLE
Error running srb init with the latest version of sorbet and parlour 4.

### ISSUE THREAD

**Issue by manhhung741** [AaronC81/parlour#110]
Error running srb init with the latest version of sorbet and parlour 4. It can be an issue with a new version of sorbet.

**To Reproduce**
Running srb init with parlour 4.

**Expected behavior**
Can run `srb init` with parlour 

**Actual behavior**

/Users/hdoan/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/sorbet-0.5.6154/lib/hidden-definition-finder.rb:151:in `write_constants': /var/folders/8_/7h7jcfm175gcx0glsb5p96mm0000gr/T/d20201213-68285-177rc95/reflection.rbi had unexpected errors. Check this file for a clue: /var/folders/8_/7h7jcfm175gcx0glsb5p96mm0000gr/T/d20201213-68285-177rc95/reflection.json.err (RuntimeError)

````
/Users/hdoan/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/parlour-4.0.1/rbi/parlour.rbi:166: The instance variable `@overridable` must be declared inside `initialize` or declared nilable https://srb.help/5005
     166 |      prop :overridable, T::Boolean
                      ^^^^^^^^^^^

/Users/hdoan/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/parlour-4.0.1/rbi/parlour.rbi:167: The instance variable `@override` must be declared inside `initialize` or declared nilable https://srb.help/5005
     167 |      prop :override, T::Boolean
                      ^^^^^^^^

/Users/hdoan/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/parlour-4.0.1/rbi/parlour.rbi:168: The instance variable `@abstract` must be declared inside `initialize` or declared nilable https://srb.help/5005
     168 |      prop :abstract, T::Boolean
                      ^^^^^^^^

/Users/hdoan/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/parlour-4.0.1/rbi/parlour.rbi:169: The instance variable `@final` must be declared inside `initialize` or declared nilable https://srb.help/5005
     169 |      prop :final, T::Boolean
                      ^^^^^
```

**Additional information**
Add any other information about the problem here. Please include the version of Parlour you're using as well as the versions of any plugins.

```
sorbet: sorbet-0.5.6154
parlour: 4.0.1
```

**Comment by jsmestad** [comment]
Hitting this too @manhhung741 did you have a workaround by chance till it get fixed?

**Comment by AaronC81** [comment]
Looks like this issue started occurring with version 0.5.6140 of Sorbet... I'll try to figure out the change which caused it.

What makes this seem strange is that the Parlour RBI works totally fine in an existing Sorbet project on that version. It's only `srb init` which runs into problems :confused: 

While not ideal, the only workaround I can think of which might be worth trying is to `srb init` the project without Parlour (or whatever depends on it, I imagine `sorbet-rails` in this case) in the Gemfile, then add that back to the Gemfile afterwards. You could then manually copy Parlour's RBI into your project to get its signatures.

**Comment by manhhung741** [comment]
Another option is using sorbet version 0.5.6139 or any version before it breaks parlour, at least until the issue is fixed on sorbet.

**Comment by manhhung741** [comment]
@AaronC81 When you know the change that causes it, can you file an issue in the sorbet repo?

**Comment by AaronC81** [comment]
@manhhung741 Sure, will do!

**Comment by AaronC81** [comment]
Opened https://github.com/sorbet/sorbet/issues/3792

**Comment by AaronC81** [comment]
Fixed by Sorbet 0.5.6183
